src/fmgc/src/flightplanning/FlightPlanAsoboSync.ts

```diff
@@ -45,142 +45,159 @@ export class FlightPlanAsoboSync {
                 Coherent.call('LOAD_CURRENT_ATC_FLIGHTPLAN').catch(console.error);
                 setTimeout(() => {
                     Coherent.call('GET_FLIGHTPLAN').then(async (data: Record<string, any>) => {
-                        console.log('COHERENT GET_FLIGHTPLAN received');
-                        const { isDirectTo } = data;
+                        console.log('COHERENT GET_FLIGHTPLAN received:');
+                        console.log('Data from MSFS flight plan:', data);
```

Comment:
Maybe comment this out for the merge

Reply:
Would like to keep this one to easier debug issue when users report problems.